src/components/pages/DataVisualizations/Graphs/CitizenshipMapSingleOffice.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+import Plot from 'react-plotly.js';
+import Table from './TableComponents/Table';
+import { colors } from '../../../../styles/data_vis_colors';
+
+const { background_color, primary_accent_color, secondary_accent_color } = colors;
+
```

Comment:
Please remove `primary_accent_color` and `secondary_accent_color` from this file if it is not being used.

Reply:
Done, thank you!

Reply:
Amazing thanks!